leetcode/src/540.c

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Could you add please a small short description of the main idea? something like that you mentioned that duplicates are places on [n even] [n odd] before single element and [n odd][n even] after and you use binary search for finding that place?

Reply:
I have already added a description to the code. I think that the format I have used is not the most professional, so if you have any advice to write the description in a better way, it would be very helpful 😄.

Reply:
It looks good for me!)